js/src/langchain.ts

```diff
@@ -0,0 +1,122 @@
+import { CallbackManager } from "@langchain/core/callbacks/manager";
+import { LangChainTracer } from "@langchain/core/tracers/tracer_langchain";
+import { Runnable, RunnableConfig } from "@langchain/core/runnables";
```

Comment:
Do we not need to import/override the storage in `@langchain/core/singletons`? Or does it not matter due to all the run tree overriding below?

Reply:
I don't think it matters, given that:
1. The handoff happens at the boundary of `@traceable` and LangChain object
2. Provided config always has precedence over ALS-one, thus the entrypoint has the mutated callback manager
3. Invocation of `AsyncLocalStorage.run()` happens when invoking RunnableLambda, which by then already has the correct callback manager to create a child run manager